perftracker/templates/_hw_lock_timeline.html

```diff
@@ -198,10 +204,12 @@ <h4 class="modal-title" id="lockLabel"></h4>
     $('#btn_show_hw_lock_dialog').click();
 
     $("#lockTitle").on("change paste keyup", function() {
-        $('#lockAction').attr("disabled", $("#lockTitle").val().trim() == "" || !$("#lockNodes :selected").length);
+        $('#lockAction').attr('disabled',
```

Comment:
make this code a inner function and use it to handle events. Code is too complicated for copy-paste